### PR TITLE
Add social menu and Twitter button back to AnswerItem

### PR DESCRIPTION
**Pull request by christianlomboy**
Added share button back to the answer items on public facing board.

### CODE REVIEW

File: client/src/components/ShareButton.js [hunk 1]
```diff
@@ -1,41 +1,141 @@
...
+  constructor(props) {
+    super(props);
+    this.state = {
+      changeColor: false,
+      isOpen: false,
+    };
+  }
+
+  componentDidMount() {
+    // window.fbAsyncInit = () => {
```
Comment (aviskarkc10) [inline client/src/components/ShareButton.js:16]: will we need this in the future?

File: client/src/components/ShareButton.js [hunk 2]
```diff
@@ -1,41 +1,141 @@
...
+    svgToChangeColor.removeEventListener('colorChanged', this.colorChangeDone);
+  }
+
+  colorChangeDone() {
+    this.setState({ changeColor: !this.state.changeColor });
+  }
+
+  changeColor() {
+    this.setState({ changeColor: !this.state.changeColor });
+  }
```
Comment (aviskarkc10) [inline client/src/components/ShareButton.js:49]: these two functions are doing the same thing. We should remove one of 'em.

**Comment by christianlomboy** [comment]
Social menu reimplementation has been pushed. Just the Twitter button in there for now.

File: client/src/components/ShareButton.js (hunk 1 above)
Comment (christianlomboy) [reply client/src/components/ShareButton.js:16]: It was added previously to anticipate having to use Facebook’s SDK in order to add a Facebook share button. Will check in with the team tomorrow to see if we still need this.

File: client/src/components/ShareButton.js (hunk 2 above)
Comment (christianlomboy) [reply client/src/components/ShareButton.js:49]: @aviskarkc10 agreed. I overlooked that

**Pull request merged**